dbcon/mysql/ha_mcs_impl_if.h

```diff
@@ -177,7 +183,8 @@ struct gp_walk_info
         inCaseStmt(false),
         cs_vtable_is_update_with_derive(false),
         cs_vtable_impossible_where_on_union(false),
-        isGroupByHandler(false)
+        isGroupByHandler(false),
+        inSubQueryLHS(0)
```

Comment:
There should be a default for inSubQueryLHSItem

Reply:
Plz use nullptr instead of 0 constant.

Reply:
Done.